pkg/daemon/rpm-ostree.go

```diff
@@ -100,38 +100,7 @@ func (r *RpmOstreeClient) loadStatus() (*rpmOstreeState, error) {
 }
 
 func (r *RpmOstreeClient) Initialize() error {
```

Comment:
Hmm, maybe it's worth removing Initialize() altogether then, since the non-rpmostree client doesn't use this either.